packages/punchcard/lib/lambda/function.ts

```diff
@@ -121,6 +121,7 @@ export class Function<T, U, D extends Dependency<any>> implements Entrypoint, Re
       const requestMapper: Mapper<T, any> = this.request === undefined ? Raw.passthrough() : Raw.forShape(this.request);
       const responseMapper: Mapper<U, any> = this.response === undefined ? Raw.passthrough() : Raw.forShape(this.response);
       return (async (event: any, context: any) => {
+        context.callbackWaitsForEmptyEventLoop = false;
```

Comment:
This could be set in the `bootstrap` function of a MongoDB database dependency.

Reply:
indeed! good idea, it's never good to lie/omit about your dependencies 👍

Reply:
It also means that the flag is only set for functions who use the dependency - better encapsulation.